core/modules/layout/tests/layout.tests.info

```diff
@@ -57,3 +63,8 @@ name = Layout Flexible Template Test
 description = Tests that flexible layout templates perform correctly.
 group = Layout
 file = layout.test
+
+; Added by Backdrop CMS packaging script on 2021-01-27
```

Comment:
We should make sure this info (added by the packaging script) doesn't accidentally end up committed to core.